core/iwasm/aot/aot_intrinsic.c

```diff
@@ -223,7 +223,7 @@ aot_intrinsic_fmin_f32(float32 a, float32 b)
     else if (isnan(b))
         return b;
     else
-        return (float32)fmin(a, b);
+        return fminf(a, b);
```

Comment:
Not very sure whether the original implementations of aot_intrinsic_fmin_f32/f64 and aot_intrinsic_fmax_f32/f64 are correct, here codes of aot_intrinsic_fmin_f32 and aot_intrinsic_fmin_f64 are inconsistent. Developer ever reported error for the implementation of interpreter:
https://github.com/bytecodealliance/wasm-micro-runtime/issues/1728
And now we have changed them to:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/interpreter/wasm_interp_classic.c#L138-L180

Suggest to use the same implementations of interpreter for aot_intrinsic_fmin_f32/f64 and aot_intrinsic_fmax_f32/f64.

Reply:
Yes, current implementations can't pass the spec test, I had fixed it but not submitted yet.
I prefer to split these fixes to small patch to simplify patch management (a patch only do one thing).

Reply:
OK, so let's merge this PR first.